tests/models/mask2former/test_modeling_mask2former.py

```diff
@@ -481,3 +482,28 @@ def test_with_segmentation_maps_and_loss(self):
             outputs = model(**inputs)
 
         self.assertTrue(outputs.loss is not None)
+
+    def test_export(self):
```

Comment:
it should be slow test -  decorated with `@slow`

Reply:
yep the testcase (the class) is already marked as slow: https://github.com/philkuz/transformers/blob/d08855c2e7e49878f8f80af9c590778f77dbe651/tests/models/mask2former/test_modeling_mask2former.py#L364

Reply:
```
tests/models/mask2former/test_modeling_mask2former.py::Mask2FormerModelIntegrationTest::test_export SKIPPED (test is slow)
```